### PR TITLE
Bump Chart version

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1333

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/prod/common/bsp/bulk-scan-orchestrator.yaml [hunk 1]
```diff
@@ -31,6 +31,12 @@ spec:
         TRANSFORMATION_URL_DIVORCE: ""
         DELETE_ENVELOPES_DLQ_MESSAGES_CRON: "0 30 0/2 * * *"
         DELETE_ENVELOPES_DLQ_MESSAGES_TTL: "PT72H"
+        TRANSFORMATION_URL_BULKSCAN: "http://bulk-scan-sample-app-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        TRANSFORMATION_URL_PROBATE: "http://probate-back-office-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        TRANSFORMATION_URL_FINREM: "http://finrem-cos-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        TRANSFORMATION_URL_SSCS: "http://sscs-bulk-scan-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        UPDATE_URL_BULKSCAN: "http://bulk-scan-sample-app-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+        UPDATE_URL_PROBATE: "http://probate-back-office-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```
Comment (nitinprabhu) [inline k8s/prod/common/bsp/bulk-scan-orchestrator.yaml:39]: I mean dynamic replacement won't work here

Comment (aliveni) [reply k8s/prod/common/bsp/bulk-scan-orchestrator.yaml]: my bad, didn't notice that. Udpated the urls now

Comment (aliveni) [reply k8s/prod/common/bsp/bulk-scan-orchestrator.yaml]: Reverted the changes, as discussed on the https://github.com/hmcts/bulk-scan-orchestrator/pull/1072

**Comment by aliveni** [comment]
> can we plz update other environments too ? perftest ithc demo ?

Updated perftest and demo. no yaml file found for ithc

**Pull request merged**